Libraries/LibWeb/CSS/StyleComputer.cpp

```diff
@@ -242,8 +243,12 @@ void FontLoader::start_loading_next_url()
         return;
     if (m_urls.is_empty())
         return;
+    auto& style_computer_realm = m_style_computer.document().realm();
```

Comment:
This and LinkElemenet feel like they shouldn't be using LoadRequest directly. Surely we should be calling fetch()?

Reply:
Yeah it should. See #2634.

Reply:
Gotcha. I won't worry about making these call sites a bit uglier in the meantime then.